client/landing/subscriptions/components/subscription-manager-page/subscription-manager-page.tsx

```diff
@@ -9,12 +11,16 @@ import './styles.scss';
 
 const SubscriptionManagementPage = () => {
 	const translate = useTranslate();
+	const { isLoggedIn } = SubscriptionManager.useIsLoggedIn();
+
 	return (
 		<>
 			<UniversalNavbarHeader
-				className="subscription-manager-header"
+				className={ classNames( 'subscription-manager-header', {
+					'is-logged-in': isLoggedIn,
+				} ) }
 				variant="minimal"
-				isLoggedIn={ false }
+				isLoggedIn={ isLoggedIn }
```

Comment:
Please note that this change is not really necessary, but I included it for correctness.

If the `isLoggedIn` prop of the `UniversalNavbarHeader` component is set to `true`, the only difference is that the header does not display the "Log in" button. But we are hiding both buttons with CSS anyways. 🙂